yt_dlp/extractor/tvp.py

```diff
@@ -483,9 +483,16 @@ class TVPVODBaseIE(InfoExtractor):
     _API_BASE_URL = 'https://vod.tvp.pl/api/products'
 
     def _call_api(self, resource, video_id, query={}, **kwargs):
-        return self._download_json(
+        is_valid = lambda x: x >= 200 and x < 300
+        is_handled_error = lambda x: x >= 400 and x < 500
```

Comment:
```suggestion
        is_valid = lambda x: 200 <= x < 300
        is_handled_error = lambda x: is_valid(x) or 400 <= x < 500
```
is_handled_error is never used by itself